presidio-anonymizer/tests/test_analyzer_result.py

```diff
@@ -0,0 +1,232 @@
+import pytest
+
+from presidio_anonymizer.entities import AnalyzerResult, InvalidParamException
+
+
+@pytest.mark.parametrize(
+    # fmt: off
+    "start, end",
+    [
+        (0, 10),
+        (10, 10),
+        (2, 8),
+        (0, 8),
+        (0, 10),
+    ],
+    # fmt: on
+)
+def test_analyzer_result_successfully_contains_another(start, end):
```

Comment:
general comment about naming of test. should be more like:
whatIsTests_WithThisCondition_ExpectingThis
so something like:

contains_with_analyzer_result_is_success/true

Reply:
Done.